scripts/generate_source.py

```diff
@@ -22,23 +22,15 @@ def RunGenerators(api: str, registry: str, targetFilter: str) -> None:
     from reg import Registry
 
     from generators.base_generator import BaseGeneratorOptions
-    from generators.dispatch_table_helper_generator import DispatchTableHelperOutputGenerator
     from generators.layer_dispatch_table_generator import LayerDispatchTableOutputGenerator
     from generators.enum_string_helper_generator import EnumStringHelperOutputGenerator
 
     # Build up a list of all generators and custom options
     generators = {
-        # TODO: vk_dispatch_table_helper.h doesn't compile.
-        # 'vk_dispatch_table_helper.h' : {
-        #    'generator' : DispatchTableHelperOutputGenerator,
-        #    'directory' : 'include/vulkan',
-        #},
-        # TODO: vk_layer_dispatch_table.h doesn't compile.
-        #
-        #'vk_layer_dispatch_table.h' : {
-        #    'generator' : LayerDispatchTableOutputGenerator,
-        #    'directory' : 'include/vulkan',
-        #},
+        'vk_layer_dispatch_table.h' : {
+           'generator' : LayerDispatchTableOutputGenerator,
+           'directory' : 'include/vulkan/layer',
```

Comment:
Since this belongs to `Vulkan::UtilityHeaders` I think the include directory should reflect that.

```suggestion
           'directory' : 'include/vulkan/utility',
```